### PR TITLE
Re-adds Poo

### PR DESCRIPTION
**Pull request by JTGSZ**
For years people have requested the coveted feature set of the poo emote, this PR adds most of what I remember about the feature from many many years ago. 
Special thanks to whoever made the saltednuts sprite.
:cl:
 * rscadd: Adds Poo.

### CODE REVIEW

**Comment by ghost** [comment]
oh no no no

**Comment by ShiftyRail** [comment]
too far

**Comment by SonixApache** [comment]
begone

File: code/game/objects/effects/decals/Cleanable/humans.dm [hunk 1]
```diff
@@ -230,3 +230,48 @@ var/global/list/blood_list = list()
...
+
+/obj/effect/decal/cleanable/literal_feces
+	name = "shit"
+	desc = "Its literally shit"
+	gender = FEMALE
+	density = 0
+	anchored = 1
+	icon = 'icons/effects/blood.dmi'
+	icon_state = "poo1"	//I just wanted it to stay on cleanable with a supercall, lol.
+	random_icon_states = list("poo1","poo2","poo3","poo4","poo5","poo6","poo7","poo8","poo9","poo10","poo11","poo12","poo13","poo14")
```
Comment (DrSnips) [inline code/game/objects/effects/decals/Cleanable/humans.dm:242]: Gee bill. Your mom lets you have 14 poo sprites?

**Comment by aceedex** [comment]
Listen here pal we don't take kindly to your kind here

**Comment by deadharold** [comment]
7 up 6 down speedmerge this right now

**Comment by ArchSunder** [comment]
im ready for april fools 2020 to be hardcore

**Comment by JTGSZ** [comment]
Maybe April Fools 2021 will be the time.

**Pull request closed without merging**